frontend/public/components/modals/delete-namespace-modal.jsx

```diff
@@ -47,6 +47,7 @@ class DeleteNamespaceModal extends PromiseComponent {
           </p>
           <input
             type="text"
+            data-test-id="project-name-input"
```

Comment:
data-cy?

Reply:
Do we want a specific `data-cy` attrib?  I don't know that we want to encode `cypress` into our html.  Seems over time we would migrate protractor to cypress anyway, so using the same `data-test-id` might be ideal?

Reply:
@benjaminapetersen `data-test-id` is not recognized by the Cypress Test Runner. It needs to be one of

```
    data-cy
    data-test
    data-testid
```

Reply:
fixed, started using `data-test`